packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/rules-1.14/etcd.yaml.patch

```diff
@@ -10,8 +10,8 @@
  apiVersion: monitoring.coreos.com/v1
  kind: PrometheusRule
  metadata:
-@@ -291,4 +292,5 @@
-         {{ toYaml .Values.defaultRules.additionalRuleLabels | nindent 8 }}
+@@ -456,4 +457,5 @@
+         {{- end }}
        {{- end }}
  {{- end }}
 +{{- end }}
```

Comment:
I think this extra {{ end }} could cause helm template errors (and therefore helm install/upgrade errors)

Reply:
Was added to fix a templating issue introduced by rebasing shenanigans